charts/spire/charts/spire-server/templates/statefulset.yaml

```diff
@@ -53,8 +53,27 @@ spec:
       securityContext:
         {{- include "spire-lib.podsecuritycontext" . | nindent 8 }}
       {{- include "spire-lib.default_cluster_priority_class_name" . | nindent 6 }}
-      {{- if or (gt (len .Values.initContainers) 0) (and .Values.upstreamAuthority.certManager.enabled .Values.upstreamAuthority.certManager.ca.create) }}
+      {{- if or (gt (len .Values.initContainers) 0) (and .Values.upstreamAuthority.certManager.enabled .Values.upstreamAuthority.certManager.ca.create) .Values.nodeAttestor.tpmDirect.enabled }}
       initContainers:
+        {{- if .Values.nodeAttestor.tpmDirect.enabled }}
+        - name: init-tpm-direct
+          securityContext:
+            {{- include "spire-lib.securitycontext" . | nindent 12 }}
+          image: {{ template "spire-lib.image" (dict "appVersion" $.Chart.AppVersion "image" .Values.nodeAttestor.tpmDirect.image "global" .Values.global) }}
+          command:
+            - sh
+            - -ec
+            - |
+              cp -a {{ .Values.nodeAttestor.tpmDirect.pluginPath }} /tpm/tpm_attestor_server
```

Comment:
Why is this step needed?

Reply:
Plugins currently have to be forked off of the spire-server/agent process so must be available within the spire-server/agent container. This makes it available for executation.

Reply:
Thanks for the explanation. Can you add that as a comment to the code?

Reply:
Thanks for adding